crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -1056,7 +1059,11 @@ pub fn extract_meshes_for_gpu_building(
                 previous_input_index,
             };
 
-            queue.push(entity, gpu_mesh_instance_builder, gpu_mesh_culling_data);
+            queue.push(
+                *render_entity,
```

Comment:
I take it this is the same as calling `render_entity.id()`?

Reply:
No, because I changed this function to take a `RenderEntity`. See the motivation in the comment below.